registry/registry.go

```diff
@@ -111,6 +123,27 @@ func getFetchModuleDirectory(paramsMap map[string]string) (fetchModuleDirectory
 	return fetchModuleDirectory, fetchModuleFile, nil
 }
 
+func getExplicitCamelCaseMapping(paramsMap map[string]string) (map[string]string, error) {
+	explicitCamelCaseMapping, ok := paramsMap[ExplicitCamelCaseMapping]
+
+	if !ok {
+		return nil, nil
+	}
+
+	mapping := make(map[string]string)
+
+	for _, mappingPair := range strings.Split(explicitCamelCaseMapping, ";") {
```

Comment:
This will allow me to call this option like:
```
proto_compiled_sources(
    name = "pb_ts_compiled_sources",
    srcs = [
        "fetch.pb.ts",
        "pms.pb.ts",
    ],
    options = {"@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway-ts:protoc-gen-grpc-gateway-ts": [
        "fetch_module_directory=go/pms/pkg/patient",
        "generate_unbound_methods=true",
        "logtostderr=true",
        "explicit_camel_case_mapping"="a1c-a1c;t2d-t2d;acuity_assessment=acuity_assessment"
    ]},
    plugins = ["@build_stack_rules_proto//plugin/grpc-ecosystem/grpc-gateway-ts:protoc-gen-grpc-gateway-ts"],
    proto = "pb_proto",
)
```